env.sh

```diff
@@ -17,6 +17,7 @@ SSL_PUBLIC_KEY="$SSL_PATH/azdis_public.crt"
 
 # Miscellaneous
 LOG_FILE="/var/log/azureDiskInspectSvc.log"
+CREDSCANSERVICEVOLUMENAME="CredScanSvc"
```

Comment:
I would suggest removing this entry.

Reply:
Looks like it is used in run.sh below.

Reply:
This is used in the run.sh script to check if the Credential Scanner is available and then to map the volume into the Disk Inspect docker container.